From 4de20ac15cc6c267b14ad2d177dd585da372e089 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 30 May 2019 18:05:18 -0500
Subject: [PATCH 3/6] Don't build GenFacades - broken package in source-build.

---
 .../Microsoft.DotNet.GenFacades.csproj                         | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/Microsoft.DotNet.GenFacades/Microsoft.DotNet.GenFacades.csproj b/src/Microsoft.DotNet.GenFacades/Microsoft.DotNet.GenFacades.csproj
index 196bc8dd..4dcad824 100644
--- a/src/Microsoft.DotNet.GenFacades/Microsoft.DotNet.GenFacades.csproj
+++ b/src/Microsoft.DotNet.GenFacades/Microsoft.DotNet.GenFacades.csproj
@@ -6,6 +6,7 @@
     <PackageType>MSBuildSdk</PackageType>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <IsPackable>true</IsPackable>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <ItemGroup>
@@ -49,4 +50,4 @@
 
   <Import Project="$(RepoRoot)eng\BuildTask.targets" />
 
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

